index.html

```diff
@@ -11,28 +12,49 @@
 
         gtag('config', 'G-KK41RCPR06');
     </script>
-    <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <script src="https://kit.fontawesome.com/31d9495df1.js" crossorigin="anonymous"></script>
-    <title>Portfolio</title>
+    <title>Aku Khan' s Portfolio</title>
     <link rel="stylesheet" href="style.css" />
-    <link rel="stylesheet" href="./assets/all.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
 </head>
 
 <body>
     <div class="main">
         <header class="heading">
-            <a class="logo" href="#"> Aku </a>
+            <a class="logo" href="#"> <h2>Aku </h2> </a>
             <nav class="mobile">
```

Comment:
- [ ]  As previously requested, here is your page at the same pixels as before.
<div class="flex">
<img src="https://user-images.githubusercontent.com/65501535/136027571-02fd2cf0-fdaa-42b2-899b-246e6ae91a24.png"  style="width: 50%;"/>
<img src="https://user-images.githubusercontent.com/65501535/136027689-8e3ee73f-0a94-47f7-8b97-8b2db8ba6c14.png"  style="width: 50%;"/>
</div>